src/app/admin/admin.component.html

```diff
@@ -1,8 +1,13 @@
 <div class="admin">
   <div class="admin__sidebar">
-    <clark-admin-sidebar></clark-admin-sidebar>
+    <clark-admin-sidebar
+      [authorizedCollections]="authorizedCollections"
```

Comment:
If the authorization of what collections should be seen is happening outside of this component, could we just call this property collections and dumb down the sidebar component to not even know the difference?